### ISSUE TITLE
Enable TAP interface for Networking

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#35]
```
Enable TAP interface for Networking

```

Original issue reported on code.google.com by `ideal2i...@gmail.com` on 26 Apr 2013 at 8:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `ideal2i...@gmail.com` on 29 Apr 2013 at 6:36
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Included in 0.9.7 Experimental
```

Original comment by `ideal2i...@gmail.com` on 5 May 2013 at 9:53
- Changed state: **Done**
